tests/unit/backend/layers/business/test_business.py

```diff
@@ -0,0 +1,802 @@
+from datetime import datetime
+import unittest
+import copy
+from unittest.mock import Mock
+from backend.corpora.common.providers.crossref_provider import CrossrefException, CrossrefProvider
+from backend.layers.business.business import BusinessLogic, CollectionQueryFilter, UserInfo
+from backend.layers.common.entities import CollectionMetadata, CollectionVersion, DatasetMetadata, DatasetStatus, Link
+from backend.layers.persistence.persistence import DatabaseProviderInterface
+from backend.layers.thirdparty.crossref_provider import CrossrefProviderInterface
+from backend.layers.thirdparty.step_function_provider import StepFunctionProviderInterface
+from tests.unit.backend.layers.persistence.persistence_mock import DatabaseProviderMock
+
+class BaseBusinessLogicTestCase(unittest.TestCase):
+
+    sample_collection_metadata = CollectionMetadata(
```

Comment:
don't use class level variables in unittests. Move these the setup or in
```
@classmethod
def setUpClass(cls)
```

Reply:
This is a bit tricky because these variables are used as default parameters in the subsequent methods. Moving them below isn't gonna allow that. Any ideas?

Reply:
move them to the global context or you can wrap them in another class called Fixtures.